package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubspot-api",
-  "version": "2.15.3",
+  "version": "2.15.4",
```

Comment:
Hi, could you please pull latest changes from master as the current version is `2.16.0` at the moment?